linetools/isgm/utils.py

```diff
@@ -886,3 +888,58 @@ def joebvp_from_components(comp_list, specfile, outfile):
     f.close()
 
 
+def unique_components(comps1, comps2, tol=5*u.arcsec):
+    """ Identify which members of comps1 are *not* within
+    comps2, to given tolerances.  Note, that repeats in
+    comps1 are not looked for.
```

Comment:
Perhaps clarify 2nd sentence? "Note: nonunique members of comps1 are not identified."

Reply:
I do not understand this either. How can a component be _within_ another component? This is based on velocity limits or was supposed to identify two components that may be considered the same physical entity?

Reply:
have tried to clarify.

Reply:
This remains unclear to me. I beleive you try to identify component duplications (?), but in this case why to allow for a tolerance? If you allow a tolerance (specially in velocity limits), then what would happen for multiple contiguous and overlapping components? Will there be considered the same? If so, why is this useful?

Reply:
Oops, I forgot to push utils.py

Look at the new docstring please